packages/astro/README.md

```diff
@@ -28,4 +28,98 @@ This package is a wrapper around `@sentry/node` for the server and `@sentry/brow
 
 ## Installation and Setup
 
-TODO
+### 1. Registering the Sentry Astro integration:
+
+Add the `sentryAstro` integration to your `astro.config.mjs` file:
+
+```javascript
+import { sentryAstro } from "@sentry/astro/integration";
```

Comment:
If I exported it from top-level `@sentry/astro`, I'd need to add it to the `index.*.js` files which still get bundled by rollup. Unless I'm missing something we could basically only turn it around, meaning the integration would be exported from top level and everything else from e.g. `@sentry/astro/config`. This might be fine for the optional `Sentry.init` files but I guess super confusing if people actually import Sentry stuff somwhere else. For example:

```js
// page-1.astro
import * as Sentry from '@sentry/astro/config';
Sentry.captureMessage('hi from component');
```

I think the current approach overall more clearly separates the unbundled integration from the rest.

Reply:
Just talked IRL, we're going to stick with `@sentry/astro/integration`